cli/azd/pkg/input/console.go

```diff
@@ -98,8 +109,12 @@ type AskerConsole struct {
 	spinnerTerminalMode     yacspin.TerminalMode
 	spinnerTerminalModeOnce sync.Once
 
-	currentIndent string
-	consoleWidth  int
+	currentIndent         string
+	consoleWidth          int
+	previewer             *progressLog
+	initialWriter         io.Writer
+	currentSpinnerMessage string
+	spinnerMutex          sync.Mutex
```

Comment:
Would be nice to add a comment to this mutex about what it guards and when folks should acquire it.